core/database/src/main/java/com/google/samples/apps/nowinandroid/core/database/dao/AuthorDao.kt

```diff
@@ -56,12 +56,8 @@ interface AuthorDao {
     /**
      * Inserts or updates [entities] in the db under the specified primary keys
      */
-    @Transaction
-    suspend fun upsertAuthors(entities: List<AuthorEntity>) = upsert(
```

Comment:
I believe we should be able to remove the now-unused `upsert` method as part of this PR.